### DISCUSSION TITLE
[Help Wanted] New mobile experience passcode typing

### DISCUSSION THREAD

**Discussion by vstepaniuk** [standardnotes/forum#2682] [General]
**Describe the issue**
Even before this, the way you entered your passcode on the iOS web was very much different from the iOS mobile app in two ways:
1. The cursor was not placed in the passcode field upon load automatically 
2. The keyboard was full sized, and not numpad 

Now these differences become very disturbing with the new mobile experience.
Please fix this!

Also, another bug, when the new mobile exp. app is opened, the iOS status bar gets white - until you place a cursor manually.

Please see more details on the hardware/software in #2176

![5C58D845-1B62-42C0-8DA5-8617DC83B65A](https://user-images.githubusercontent.com/24701640/194901416-1e4b3529-bb2e-4d44-818f-086ba10bcd25.png)

**Comment by effieeee** [comment]
Hi @vstepaniuk indeed the passcode lock doesn't have those particular settings at the moment for the new mobile view, so I'll be marking this as an enhancement suggestion for the new mobile app!

**Comment by pranabdas** [comment]
@effieeee On a small screen device (original iPhone SE), the text and buttons do not appear centered.

![Photo 12-10-22, 5 37 13 PM](https://user-images.githubusercontent.com/31024886/195308066-91f1a8b4-0117-4b09-8dd7-e0bfc1d4eead.png)

**Comment by effieeee** [comment]
I see, just to clarify though, do you have any **Accessibility** options set for your device, such as a larger font size or special zoom functions?

**Comment by pranabdas** [comment]
No, I have default text size and zoom level. Thank you.

**Comment by pranabdas** [comment]
The off-centered lock screen issue on small screen devices is currently resolved for me. Thank you 🙏

**Comment by DecemberGiraffe** [comment]
Have the cursor issue on Android as well
